libs/blocks/aside/aside.css

```diff
@@ -370,19 +448,84 @@
   justify-content: flex-start;
 }
 
+.aside.split .foreground.container .text .icon-stack-area {
+  display: flex;
+  flex-flow: row wrap;
+  flex-direction: column;
+  gap: 12px;
+  margin: -8px 0 var(--spacing-s);
+  width: 100%;
+  padding: 0;
+  list-style-type: none;
+}
+
+.aside.split .foreground.container .text .icon-stack-area li,
+.aside.split .foreground.container .text .icon-stack-area li a {
```

Comment:
specificity is starting to get pretty deep. Is it possible to remove some selectors, for example : `.aside.split .icon-stack-area li`

Reply:
@JasonHowellSlavin I have made the suggested changes